### PR TITLE
[MIG][10.0]stock analytic

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
Stock Analytic
==============
Adds an analytic account in stock move to be able to get analytic information when generating the journal items.

Additional features:
- Added the analytic account in the search view for the stock moves
- Remove the analytic account in the stock move in the inventory view. The analytic account is always empty, so it makes not sense to be there. If the analytic experts agree I'd propose another module to do inventory adjustments for a specific analytic account. It's already implemented.

### CODE REVIEW

**Comment by AaronHForgeFlow** [comment]
@Eficent

**Comment by AaronHForgeFlow** [comment]
:zzz:

**Comment by LoisRForgeFlow** [comment]
Runbot is down :cry:

File: stock_analytic/README.rst [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+------------
+
+* Hanane ELKHAL <hanane@julius.fr>
+* Yvan Patry <yvan@julius.fr>
+* Pierre <pierre@julius.fr>
+* Mathieu VATEL <mathieu@julius.fr>
+* Fabio Vílchez <fabio.vilchez@clearcorp.co.cr>
+* Andhitia Rama <andhitia.r@gmail.com>
+* Michael Viriyananda <viriyananda.michael@gmail.com>
+* Eficent <contact@eficent.com>
```
Comment (LoisRForgeFlow) [inline stock_analytic/README.rst]: better to put yoursefl, right?

File: stock_analytic/README.rst [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+Assigned Journal Items created from Stock Move with Analytic Account
+--------------------------------------------------------------------
+
+If stock move automatically create journal entry, the journal entry will
+contain journal items with following rule:
+
+#. Journal item with account equal to product's valuation account will not be
+   assigned any analytic account
+#. Journal item with account not equal to product's valuation account will be
+   assigned analytic account according to stock move's analytic account
```
Comment (LoisRForgeFlow) [inline stock_analytic/README.rst]: These two sentences are not very clear IMHO

**Review by LoisRForgeFlow — COMMENTED** [review]
first code review

File: stock_analytic/README.rst (hunk 1 above)
Comment (AaronHForgeFlow) [reply stock_analytic/README.rst]: But it was like that in v9 so not sure

File: stock_analytic/README.rst (hunk 2 above)
Comment (AaronHForgeFlow) [reply stock_analytic/README.rst]: You're right I'll rewrite this

**Review by LoisRForgeFlow — APPROVED** [review]
Functional test, works like a charm! :fire:

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**